### PR TITLE
[BLOG] Add but-what-are-traitlets

### PR DESCRIPTION
**Pull request by MarsBarLee**
[Link to blog post but-what-are-traitlets.md, successful build.](https://labs-git-but-what-are-traitlets-quansight.vercel.app/blog/but-what-are-traitlets)

This blog post is being resubmitted with the new workflow:
- Branching directly from the Quansight-website repository instead of a fork
- which allows me to authorize deployment of Vercel to build preview URLs
- making it easier to catch errors and make changes as necessary

The previous blog post PR for but-what-are-traitlets, #367, is closed.

## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

## Links to other Quansight blogs:

no labs links found
no llc links found

### CODE REVIEW

File: apps/labs/posts/but-what-are-traitlets.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
+---
+title: 'Traitlets - an introduction & use in Jupyter configuration management'
+published: August 30, 2020
+author: matthias-bussonnier
+description: 'You have probably seen Traitlets in applications, you likely even use it.
+The package has nearly 5 million downloads on conda-forge alone. But, what is Traitlets?
+In this post we'll answer this question along with where Traitlets came from,
```
Comment (MarsBarLee) [inline apps/labs/posts/but-what-are-traitlets.md:7]:
```suggestion
In this post we''ll answer this question along with where Traitlets came from,
```
Address Vercel build error `error YAMLException: can not read a block mapping entry; a multiline key may not be an implicit key at line 9, column 9`, updated single quotes to properly close YAML description

**Comment by MarsBarLee** [comment]
[Link to blog post but-what-are-traitlets.md, successful build.](https://labs-git-but-what-are-traitlets-quansight.vercel.app/blog/but-what-are-traitlets)

**Comment by MarsBarLee** [comment]
@noatamir Ready for review. [Direct link to blog post's successful build for but-what-are-traitlets.md](https://labs-git-but-what-are-traitlets-quansight.vercel.app/blog/but-what-are-traitlets). Hero and feature image loads.

File: apps/labs/posts/but-what-are-traitlets.md [hunk 2]
```diff
@@ -0,0 +1,351 @@
+---
+title: 'Traitlets - an introduction & use in Jupyter configuration management'
+published: August 30, 2020
+author: matthias-bussonnier
+description: 'You have probably seen Traitlets in applications, you likely even use it.
+The package has nearly 5 million downloads on conda-forge alone. But, what is Traitlets?
+In this post we''ll answer this question along with where Traitlets came from,
+its applications, and a bit of history.'
+category: [PyData Ecosystem]
```
Comment (gabalafou) [inline apps/labs/posts/but-what-are-traitlets.md:9]:
```suggestion
category: [PyData ecosystem]
```

**Review by gabalafou — APPROVED** [review]

**Pull request merged**